README.md

```diff
@@ -1,150 +1,10 @@
 # Digital Humanities Final
 
-![Title Image](assets/images/intro-image.png)
+![Home Image](assets/images/Wager's_Action_off_Cartagena,_28_May_1708.jpg)
 
-## Instructor and Author
+The transatlantic slave trade, also known as triangle trade, was a product of the colonization and expansion of imperialistic European nations searching for new economic opportunities in the Americas. In most cases, goods would be shipped from Europe to Africa, slaves from Africa to the Americas, and goods from the Americas back to Europe, thus completing the triangle of trade. However, long slave voyages sometimes encountered problems associated with piracy. 
 
-[David J. Thomas](mailto:dave.a.base@gmail.com), [thePortus.net](http://thePortus.net)<br />
-Instructor of Ancient History and Digital Humanities<br />
-Department of History<br />
-[University of South Florida](https://github.com/usf-portal)
+Piracy have a long history dating back more then 2000 years, according to historian Alfred Bradford (1). Ravaging the seas in search of vast fortunes, pirates sealed their place in history as ruthlessly violent savages. One of the most popular eras in pirate history, an era of fascination among historians and the general public partially because of its place in popular culture, is known as the Golden Age (c.1700-c.1730). 
```

Comment:
First, separate from what follows: did pirates really seal their own place in history, or have others controlled the images of them?

You cited exactly as I told you to, good. I've been thinking on it, trying to come up with a better citation system, and I think I have a solution...

* We should make the citation clickable, so that it at least can take the reader to the bibliography page quickly.
* I think the number by itself would be confusing so maybe something like (Bibliography 1) works a bit better
* But I also think that in the case of monographs like Bradford's, you should probably provide a specific page or range of pages, since that is tied to the claim of 2000 years. If you wanted to do that you could do it like (Bibliography 1: 123-124)

If you wanted to do that, with a link to the bibliography, you could type either of the following examples. The first makes everything inside the parentheses linked (generally looks better), the second makes even the parentheses linked (probably easier to understand).

```markdown
([Bibliography 1: 123-124](bibliography.md))

[(Bibliography 1: 123-124)](bibliography.md)
```

Reply:
Another option I thought of... if the source is available online, you can just link to it, even if you put it in your bibliography.